src/fontra/views/editor/edit-tools-shape.js

```diff
@@ -59,12 +59,12 @@ export class ShapeToolRect extends BaseTool {
       const drawPath = new Path2D();
       this.drawShapePath2D(drawPath, mouseRect, event);
       this.sceneModel.shapeToolShapePath = drawPath;
-      this.sceneModel.event = event;
+      this.sceneModel.shapeToolCtrlKey = event.ctrlKey;
```

Comment:
I would name this after what this flag is supposed to do, say `.shapeToolShowFill`.

Reply:
I changed it.